dsms/knowledge/webform.py

```diff
@@ -0,0 +1,68 @@
+"""Webform model"""
+
+from typing import Any, List, Optional
+
+from pydantic import BaseModel, ConfigDict, Field
+from pydantic.alias_generators import to_camel
+
+
+class WebformSelectOption(BaseModel):
+    """Choices in webform"""
+
+    label: Optional[str] = Field(None)
+    value: Optional[Any] = Field(None)
+    disabled: Optional[bool] = Field(False)
+
+
+class Inputs(BaseModel):
+    """Input fields in the sections in webform"""
+
+    model_config = ConfigDict(
+        alias_generator=lambda field_name: to_camel(  # pylint: disable=W0108
+            field_name
+        )
+    )
+
+    id: Optional[str] = Field(None)
+    label: Optional[str] = Field(None)
+    widget: Optional[str] = Field(None)
+    default_value: Optional[Any] = Field(None)
+    value: Optional[Any] = Field(None)
+    choices: List[WebformSelectOption] = Field([])
+    on_change: Optional[Any] = Field(None)
+    check: Optional[Any] = Field(None)
+    error: Optional[str] = Field(None)
+    feedback: Optional[str] = Field(None)
+    hint: Optional[str] = Field(None)
+    measurement_unit: Optional[str] = Field(None)
+    mapping: Optional[str] = Field(None)
+    mapping_name: Optional[str] = Field(None)
```

Comment:
I am missing the `mapping_unit` here.

Reply:
I couldn't find any field named mapping_unit in the ktype service.
[https://gitlab.cc-asp.fraunhofer.de/dsms/webform-builder/-/blob/main/projects/webform-builder/src/lib/models/webform-input.ts?ref_type=heads#L14](url)